### PR TITLE
Update gateway_mfr_rs v0.2.1

### PR DESCRIPTION
**Pull request by posterzh**
**Issue**

- Link: https://github.com/NebraLtd/hm-pyhelper/issues/164
- Summary:
To align with upstream gateway_mfr_rs version, upgrade the gateway_mfr executable embedded in hm-pyhelper package safely from v0.1.7 to v0.2.1. 

**How**
- Upgrade the gateway_mfr executable embedded in `hm-pyhelper` from v0.1.7 to v0.2.1. 
- Update [run_gateway_mfr](https://github.com/NebraLtd/hm-pyhelper/blob/2f654d7b635c14ea2726da924d43e4febf303f21/hm_pyhelper/miner_param.py#L27) in `hm-pyhelper` to accomodate this change.
- Bump `hm-pyhelper` version in `hm-diag`.

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: settings.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [versions]
-FIRMWARE_VERSION=2022.06.09.1
-DIAGNOSTICS_VERSION=22a67fa
+FIRMWARE_VERSION=2022.06.09.2
```
Comment (shawaj) [inline settings.ini:2]: I think we had previously done -2

Comment (shawaj) [reply settings.ini:2]: The way we do it is to take the Helium release value add is, and if we do all interim change we add -1, -2, -3 Etc etc

In this case,.2022.06.09.0 was the Helium version, you then did an update which should have had it as 2022.06.09.0-1 but you did as 2022.06.09.1

So this one should be 2022.06.09.0-2

Unless @marvinmarnold wanted to change this?

Comment (posterzh) [reply settings.ini:2]: Make sense. Thank you.

Comment (posterzh) [reply settings.ini]: `2022.06.09.0-2` is not taken and looks reasonable. Updated!

**Review by pritamghanghas — APPROVED** [review]

**Pull request merged**